### ISSUE TITLE
No file diectory issue

### ISSUE THREAD

**Issue by vivek-ins** [terraform-google-modules/terraform-example-foundation#833]
### TL;DR

Error: External Program Execution Failed
│
│   with module.build_terraform_image.data.external.env_override[0],
│   on .terraform/modules/build_terraform_image/main.tf line 73, in data "external" "env_override":
│   73:   program = ["${path.module}/scripts/check_env.sh"]
│
│ The data source received an unexpected error while attempting to execute the program.
│
│ Program: .terraform/modules/build_terraform_image/scripts/check_env.sh
│ Error: fork/exec .terraform/modules/build_terraform_image/scripts/check_env.sh: no such file or directory

### Terraform Configuration

```hcl
module "bootstrap_csr_repo" {
  source                = "terraform-google-modules/gcloud/google"
  version               = "~> 3.1.0"
  upgrade               = false
  
  create_cmd_entrypoint = "${path.module}/scripts/push-to-repo.sh"
  create_cmd_body       = "${module.tf_source.cloudbuild_project_id} ${split("/", module.tf_source.csr_repos[local.cloudbuilder_repo].id)[3]} ${path.module}/Dockerfile"
}
```

### Terraform Version

```sh
1.3.1
```

**Comment by bharathkkb** [comment]
Thanks for the report @vivek-ins 
From the error it looks like `.terraform` dir may not exist. Could you try rerunning `terraform init` and checking if `.terraform` dir exists?

**Comment by obriensystems** [comment]
Verified 0-bootstrap - working fine with the normal terraform init/plan/apply
see #940
